Documentation/Configuration/ConfigurationOverview.rst

```diff
@@ -25,14 +25,14 @@ Configuration overview: files
 Global files
 ------------
 
-:file:`<webroot>/typo3conf/LocalConfiguration.php`:
+:file:`config/system/settings.php`:
```

Comment:
As this is an important page for starters we should mention the two locations "config/system/settings.php" and "typo3conf/system/settings.php".

Also below. I know it is not done with "config/sites" below, but this can be adjusted too. What do you think?

Reply:
I would suggest to do that in a follow-up as the page needs some love anyway?

Reply:
Go ahead and please create a ticket so it does not gets lost.

Reply:
Created one: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/2288